.github/actions/opa/action.yml

```
@@ -0,0 +1,39 @@
+name: 'OPA CI'
```

Why change workflow name?
```suggestion
Test OPA Policies
```

---

This is mainly because it's not just testing; it involves fmt check, bundle building, test, opa check, and lint with Regal. So, I thought `CI` better describes the steps.

But I could change it to `Test OPA Policies`.